amacc.c

```diff
@@ -239,6 +240,10 @@ enum {
 
     SYSC, /* 30 system call */
     CLCA, /* 31 clear cache, used by JIT compilation */
+
+    PD , /* 32 Hide next instruction from peephole opt */
```

Comment:
I am afraid that the naming scheme `PD` and `P0` would not be meaningful mnemonic.

Reply:
See the documentation. :) 
PD -- peephole disable, P0 -- Note to peephole optimizer that a function returns an R0 value at this location.

Reply:
How about `PHD` for disabling peephole optimization and `PHE` for enabling?

Reply:
I used to take that approach. I was putting a markrker at the beginning and end of function calls.  In fact, it was just when I picked this back up again two weeks ago that I had an epihany that disabling one instruction was the cleanest way to do things.  When you get into the details of implementing this, you realize that there are a lot of terrible things that can go wrong, and that a surgical knife works better than a machete. Using "P0" to indicate that the hardware R0 now contains a "live" value is also a one off that is cleanest.

Reply:
PS  I had been using CB and CE to mean comment begin and comment end.

Reply:
To be more consistent, I would expect the prefix `PH` enforced on the naming. That is, `PHCB` and `PHCE`.